apps/mobile/src/services/sign-up.ts

```diff
@@ -0,0 +1,7 @@
+import { UserType } from '@animavita/types';
+
+import client from './http-client';
+
+export const signUp = (user: UserType) => {
```

Comment:
`UserType` doesn't seem to be the correct one to be used here. WDYT about creating a new `sign-up-dto.ts` file and a new `SignUpRequest` type in it? Similar to what we have in the `sign-in-dto` file.

Reply:
Sure! I was gettin' type mismatch error and was about to ask to do the same. Thank you 😄